src/styles/components/_profile-card.scss

```diff
@@ -0,0 +1,109 @@
+@import '../abstracts/_abstracts-dir';
+
+.profile-card {
+  display: flex;
+  justify-content: flex-end;
+
+  @include respond-above('md') {
+    width: 30%;
+  }
+
+  &--background-image {
+    background: {
+      image: url('/images/blue-ellipse.png');
+      repeat: no-repeat;
+      size: contain;
+      position: center;
+    }
+  }
+
+  &__body {
+    flex-direction: column;
+    box-sizing: border-box;
+    box-shadow: 0 0 rem(20px) 0 box-shadow-opacity(0.16);
+    width: fit-content;
+    padding: rem(12px 12px 16px);
+    margin: rem(24px) 0;
+    background-color: $light;
+
+    @include flex-center();
+
+    @include respond-above('sm') {
+      padding: rem(16px 16px 24px);
+    }
+
+    @include respond-above('lg') {
+      padding: rem(20px 20px 28px);
+    }
+
+    @include respond-above('xl') {
+      padding: rem(24px 24px 32px);
+    }
+  }
+
+  &__image {
+    width: rem(80px);
+    height: rem(70px);
+    margin-bottom: rem(12px);
+
+    @include respond-above('sm') {
+      width: rem(120px);
+      height: rem(100px);
+    }
+
+    @include respond-above('md') {
+      width: rem(160px);
+      height: rem(140px);
+    }
+
+    @include respond-above('lg') {
+      width: rem(200px);
+      height: rem(180px);
+    }
+
+    @include respond-above('xl') {
+      width: rem(230px);
+      height: rem(210px);
+    }
+  }
+
+  &__label {
+    &:not(:last-child) {
+      margin-bottom: rem(4px);
+
+      @include respond-above('md') {
+        margin-bottom: rem(8px);
+      }
+    }
+
+    &--name {
```

Comment:
How about renaming this modifier to `large` and `small` for the other one?

Reply:
Since we only have 2 modifiers for this label. How about `primary` and `secondary` ?